uwrt_mars_rover_neopixel/CMakeLists.txt

```diff
@@ -36,12 +36,14 @@ catkin_package(
 include_directories(
         include
         ${catkin_INCLUDE_DIRS}
+        ../uwrt_mars_rover_hw_bridge_pkg/uwrt_mars_rover_hw_bridge/config
```

Comment:
This is not the proper way to add a library from another ros package. Read this: https://roboticsbackend.com/ros-include-cpp-header-from-another-package/